content/source/docs/enterprise/private/rhel-install-guide.html.md

```diff
@@ -11,20 +11,27 @@ This install guide is specifically for users of Private Terraform Enterprise ins
 ## Install Recommendations
 
 * RedHat Enterprise Linux version 7.3 or 7.4
-* Docker CE or EE version 17.06 or later. These versions are not available in the standard RHEL yum repositories.
+* Docker 1.13.1 (available in RHEL extras) or Docker CE or EE version 17.06 or later. The later versions are not available in the standard RHEL yum repositories.
    * For Docker CE, you can use the official CentOS docker repository. Instructions are available here: https://docs.docker.com/install/linux/docker-ce/centos/#install-from-a-package.
    * For Docker EE, there are explicit RHEL instructions to follow: https://docs.docker.com/install/linux/docker-ee/rhel/ 
 * A properly configured docker storage backend, either:
    * Devicemapper configured for production usage, according to the Docker documentation: https://docs.docker.com/storage/storagedriver/device-mapper-driver/#configure-direct-lvm-mode-for-production. This configuration requires a second block device available to the system to be used as a thin-pool for Docker. You may need to configure this block device before the host system is booted, depending on the hosting platform.
    * A system capable of using overlay2. The requires at least kernel version 3.10.0-693 and, if XFS is being used, the flag ftype=1. The full documentation on this configuration is at: https://docs.docker.com/storage/storagedriver/overlayfs-driver/
 
+## Mandatory Configuration
+
+If you opt to use Docker from RHEL extras, then you must make a change to it's default configuration to avoid hitting an out of memory bug.
+
+1. Open `/usr/lib/systemd/system/docker.service`
+1. Remove the line that contains `--authorization-plugin=rhel-push-plugin`
+1. Run `systemctl daemon-reload && systemctl restart docker`
+1. Run `docker info 2> /dev/null | grep Authorization` to verify that there are no authorization plugins active.
+   If nothing is printed, your installation is properly configured. If anything is printed, please
+   contact support for further assistance.
```

Comment:
Should we add a link to the Moby issue and say that RedHat are working on fixing it upstream and to contact them or follow the issue for the state of the update?

Reply:
I'd say no, we should revise this page when that fix is released instead.

Reply:
👍